src/views/RecordingsView.vue

```diff
@@ -219,6 +223,21 @@ export default {
     },
     getResultsDisplayStyle() {
       return this.showCards ? "card" : "row";
+    },
+    viewRecordingQueryString() {
+      const queryCopy = JSON.parse(JSON.stringify(this.serialisedQuery));
+
+      // delete date data as we use this to find the next/previous recordings
+      delete queryCopy.days;
+      delete queryCopy.from;
+      delete queryCopy.to;
+
+      // delete type as this is controlled by the view
+      delete queryCopy.type;
+      delete queryCopy.limit;
+      delete queryCopy.offset;
+
+      return querystring.stringify(queryCopy);
```

Comment:
Do we really need this `querystring` dependency rather than just using the built-in `encodeURIComponent`? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent